guides/releasing/Release Workflow.md

```diff
@@ -0,0 +1,328 @@
+- Open the last step called `Build and upload release tarball` and copy the url to the tarball that was uploaded to `exp-artifacts` S3 bucket.
+- Now go to `expo/turtle` repo and put the copied link into `shellTarballs/<platform>/sdkXX` file, where `<platform>` is the shell app platform you're updating.
+- Put appropriate change information in the `CHANGELOG.md` file, commit and then push changes.
+
+## 5.3. Make adhoc client shell app
+
+**Why:**
+
+**How:**
+
+## 5.4. Deploy Turtle with new shell tarballs
+
+| Requirements                                                         |
+| -------------------------------------------------------------------- |
+| [5.2. Make shell app build](#5-2-make-shell-app-build)               |
+| [5.3. Make adhoc client shell app](#5-3-make-adhoc-client-shell-app) |
+
+**Why:** Once we've made standalone and adhoc client shell apps, we're now ready to deploy Turtle to staging, test it and then roll out to production.
+
+**How:**
+
+- Release Turtle to staging by updating its submodule in `expo/universe` repo. CI is releasing it automatically on `master` branch.
+- When the new shellTarball is successfully deployed to staging, try to build a standalone app by running `EXPO_STAGING=1 expo build:<platform>`.
+- If everything seems to be fine, deploy Turtle to production by approving `turtle_<platform>_approve_production` job in `turtle_<platform>` workflow.
+
+# Stage 6 - Final release
+
+## 6.1. Publishing final packages to NPM registry
+
+**Why:** Previously we've published prereleased versions of packages and now, if everything works like a charm, we can publish final (stable) versions of those packages.
+
+**How:**
+
+- On the release branch, use an expotool that will do all of the needed work - `et publish-packages`. You already used it to publish prereleased packages. Now we can publish them as stable releases and tag them as `latest` which is the default behavior of that script.
+
+## 6.2. Publishing final project templates
+
+| Requirements                                                                                     |
+| ------------------------------------------------------------------------------------------------ |
+| [6.1. Publishing final packages to NPM registry](#6-1-publishing-final-packages-to-npm-registry) |
+
+**Why:** Once the final packages are out, we can now make a final version of project templates as well.
+
+**How:**
+
+- On a new branch, check all `expo-template-*` packages under `templates` directory and bump dependencies versions wherever possible. Use versions stored in `packages/expo/bundledNativeModules.json` for vendored libs like `react-native-gesture-handler`.
+- Run `et publish-templates` and answer to questions it asks.
+- Create a pull request from your branch to `master`. Make sure a reviewer will cherry-pick that commit to the release branch as well.
+
```

Comment:
Publishing new docs needs to be a step here, with a substep of updating the RN docs if we've updated the RN version this release.